### PR TITLE
Feature/add statistics types

### PR DESCRIPTION
**Pull request by Esteban100k**
Add Types in BlockchainDatacharts and CO2 Emissions

### CODE REVIEW

File: src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx [hunk 1]
```diff
@@ -2,18 +2,21 @@ import React from 'react'
...
-const CountriesBarMeter = ({ darkMode, titleText, dataSeries }) => {
-    const sortByAndLoadBar = data => {
-        let sortedData = sortBy(data, o => -o.Value)
-        let dataChart = sortedData.map((dat, index) => {
+const CountriesBarMeter = ({ darkMode, titleText, dataSeries }: Meter) => {
+    const sortByAndLoadBar = (data: Value): Data[] => {
+        // @ts-ignore:next-line
+        let sortedData: Value[] = sortBy(data, object => object && -object.Value)
+        let dataChart = sortedData.map((dat, index): Data => {
+            const country = dat.Country || ''
```
Comment (Chriserus) [inline src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx]: `dat` should be renamed to `data`

File: package-lock.json
Comment (Chriserus) [inline package-lock.json:1]: IMO we shouldn't have `package-lock.json`, as we're using yarn and there is already `yarn.lock` file

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx (hunk 1 above)
Comment (danilocha) [reply src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx]: Hi @Chriserus, we already changed the dat for data

File: package-lock.json
Comment (danilocha) [reply package-lock.json:1]: I already deleted the package.lock

**Review by Chriserus — APPROVED** [review]

**Pull request merged**